ModelCatalogueCorePlugin/grails-app/assets/javascripts/modelcatalogue/core/ui/catalogueElementTreeviewItem.coffee

```diff
@@ -180,6 +188,19 @@ angular.module('mc.core.ui.catalogueElementTreeviewItem', ['mc.util.names', 'mc.
         $scope.collapseOrExpand()
         $scope.treeview.select(element)
 
+      $scope.metadataOccurrencesToAsterisk = (element) ->
+        if element.$$metadata?.values? && element.getElementTypeName() == 'Data Element'
```

Comment:
@musketyr it would probably make sense to display this only for data elements which are just below the data class in tree, right (see https://metadata.atlassian.net/browse/MET-792). I don't have here a element.$$parentInTree how and where would I store this attribute to be able to use it here in this place. (my guess is line 101 but I have no clue how :goberserk:  :-D)